.circleci/config.yml

```diff
@@ -86,11 +86,11 @@ jobs:
       - run:
           name: install pyenv
           command: |
-            brew install pyenv
+            HOMEBREW_NO_AUTO_UPDATE=1 brew install pyenv
```

Comment:
Since you added inline env var def here, you can remove it from the `environment` section above.